trestle/core/ssp_io.py

```diff
@@ -302,8 +302,6 @@ def get_control_response(
                     header = f'Implementation for part {label}'
                     md_writer.new_header(1, title=header)
                 for comp_name, comp_response in response_per_component.items():
-                    if comp_name == SSP_MAIN_COMP_NAME:
-                        continue
```

Comment:
The difference between this and the pre - #1358 behavior is that `Component: This System` will be output for each part now, where it was skipped earlier.

Reply:
Is there/should there be a corresponding test case?

Reply:
I converted this to a draft. I dug in on the existing test some more and realized that it's trading one problem for another: instead of missing `This System` statements for controls with parts, it's now doubling up `This System` statements for controls without parts...

I'll give it another shot along with a test that fails before the change and passes afterwards.

Reply:
Thanks @rahearn. I did not catch that during my testing of this change.